### DISCUSSION TITLE
Trouble with the REM

### DISCUSSION THREAD

**Discussion by raymop** [tagyoureit/nodejs-poolController#472] [App and O/S Config Questions] [answered]
Started having this problem today.  Everything was working well last night.
But, now the REM throws an error.
I have pm2 stop all in effect.
Any thoughts?

pi@raspberrypi:~/relayEquipmentManager $ npm run start:cached

> relay-equipment-manager@7.5.0 start:cached /home/pi/relayEquipmentManager
> node dist/app.js

linux
info: Starting up SSDP server
info: Initializing GPIO Pins 0
info: Initializing SPI Bus #0
info: SPI Bus #0 Initialized
info: Initializing SPI Bus #1
info: SPI Bus #1 Initialized
info: Initializing i2c Interface
info: Detecting i2c Buses
info: Detecting i2c Buses /sys/class/i2c-dev/i2c-1
info: Detecting i2c Buses /sys/class/i2c-dev/i2c-20
info: Detecting i2c Buses /sys/class/i2c-dev/i2c-21
info: Initializing 1-Wire Interface
info: Detecting 1-Wire Buses
info: oneWireInterface: w1_gpio                16384  0
wire                   36864  1 w1_gpio

info: Detecting 1-Wire Buses at /sys/bus/w1/devices
info: Initializing Generic Devices
info: Initializing i2c Bus #1
info: Device 10k Temperature Probe initialized for generic device 4 - AIR TEMP
info: Device 10k Temperature Probe initialized for generic device 5 - POOL TEMP
info: Generic Devices Initialized
Error: listen EADDRINUSE: address already in use 0.0.0.0:8080
    at Server.setupListenHandle [as _listen2] (net.js:1331:16)
    at listenInCluster (net.js:1379:12)
    at doListen (net.js:1516:7)
    at processTicksAndRejections (internal/process/task_queues.js:83:21)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! relay-equipment-manager@7.5.0 start:cached: `node dist/app.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the relay-equipment-manager@7.5.0 start:cached script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/pi/.npm/_logs/2022-06-04T19_10_47_713Z-debug.log

**Comment by rstrouse** [comment, marked as answer]
You are trying to start REM and another copy is already running.  Do you have a copy started with PM2 or npm &start?

**Reply by raymop** [reply]
Yep, I did.  Thx.

**Comment by rstrouse** [comment]
Type pm2 monit and you will see REM running.  If you do not have REM running some other application is now trying to occupy TCP port 8080.  Btw do you really have 3 i2c buses present on your system?

**Reply by raymop** [reply]
I did a restart from scratch and everything looks good, except for the air temp.
